package.json

```diff
@@ -16,19 +16,21 @@
   "readmeFilename": "README.md",
   "dependencies": {
     "@atlaskit/avatar": "4.0.5",
-    "@atlaskit/button": "3.0.0",
+    "@atlaskit/button": "5.4.2",
     "@atlaskit/button-group": "1.1.3",
-    "@atlaskit/dropdown-menu": "1.4.0",
+    "@atlaskit/dropdown-menu": "3.10.2",
     "@atlaskit/field-text": "2.7.0",
-    "@atlaskit/flag": "3.0.0",
+    "@atlaskit/flag": "6.1.0",
     "@atlaskit/icon": "7.0.0",
-    "@atlaskit/inline-dialog": "3.2.0",
+    "@atlaskit/inline-dialog": "5.0.2",
     "@atlaskit/inline-message": "2.1.1",
-    "@atlaskit/modal-dialog": "2.1.2",
+    "@atlaskit/modal-dialog": "2.6.0",
     "@atlaskit/multi-select": "6.2.0",
     "@atlaskit/spinner": "2.2.3",
     "@atlaskit/tabs": "2.0.0",
-    "@atlaskit/toggle": "2.0.4",
+    "@atlaskit/theme": "2.2.0",
+    "@atlaskit/toggle": "2.6.1",
+    "@atlaskit/tooltip": "6.0.0",
```

Comment:
I can try to separate css changes with all other changes. It won't work to only commit updates because they were breaking quite a lot of stuff with class name changes, property changes, missing icons...